packages/cli/lib/projects.js

```diff
@@ -372,9 +375,19 @@ const makePollTaskStatusFunc = ({
 
     spinnies.add('overallTaskStatus', { text: 'Beginning' });
 
-    const initialTaskStatus = await statusFn(accountId, taskName, taskId);
+    const [
+      initialTaskStatus,
+      { topLevelComponentsWithChildren: taskStructure },
+    ] = await Promise.all([
+      statusFn(accountId, taskName, taskId),
+      structureFn(accountId, taskName, taskId),
```

Comment:
Just making sure. Will this be backwards compatible for accounts that are not using the dependent subcomponents BE yet?

Reply:
Yep, it's backwards compatible in the same way that the build/deploy details pages are (the structure endpoint will just return all components as top level on older projects). Just need to make sure that gets deployed on prod before we release this, so might not make sense to merge this for a bit.